client/components/masonry-wave/index.jsx

```diff
@@ -0,0 +1,66 @@
+import './style.scss';
+import { useDesktopBreakpoint } from '@automattic/viewport-react';
+
+// fills the columns with images LTR to imitate the CSS grid behavior
+const fillColumnsWithImages = ( amountOfColumns, images ) => {
+	const columns = Array( amountOfColumns );
+	let columnIndex = 0;
+
+	images.forEach( ( image ) => {
+		if ( columnIndex === amountOfColumns ) {
+			columnIndex = 0;
+		}
+
+		if ( ! columns[ columnIndex ] ) {
+			columns[ columnIndex ] = [];
+		}
+
+		columns[ columnIndex ].push( image );
+
+		columnIndex++;
+	} );
+
+	return columns;
+};
+
+const MasonryWaveElement = ( props ) => {
+	const { image } = props;
+
+	return (
+		<img src={ image?.src } alt={ image?.alt ?? 'alt text' } className="masonry-wave__element" />
+	);
+};
+
+const MasonryWaveColumn = ( props ) => {
+	const { images } = props;
+
+	return (
+		<div className="masonry-wave__column">
+			{ images.map( ( image, index ) => (
+				<MasonryWaveElement key={ `masonry-wave__row-${ index }` } image={ image } />
+			) ) }
+		</div>
+	);
+};
+
+const MasonryWave = ( props ) => {
```

Comment:
Sorry for being super late here 😅... Is this component still used? Should we remove it if it's not?

If it's still used, can we addd `readme`, `example`, and integrate it in devdocs https://wpcalypso.wordpress.com/devdocs/? Also, I feel that the component name `MasonryWave` doesn't really help in understanding what this component is used for. Is it for rendering an image grid?